pycbc/waveform/compress.py

```diff
@@ -274,7 +280,8 @@ def compress_waveform(htilde, sample_points, tolerance, interpolation,
     logging.info("mismatch: %f, N points: %i (%i added)" %(mismatch,
         len(comp_amp), len(added_points)))
     
-    return CompressedWaveform(sample_points, comp_amp, comp_phase,
+    return CompressedWaveform(amp_sample_points, phase_sample_points,
+                comp_amp, comp_phase,
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->